x/wasm-storage/keeper/keeper.go

```diff
@@ -158,6 +148,43 @@ func (k Keeper) GetAllWasms(ctx sdk.Context) []types.Wasm {
 	return wasms
 }
 
+// WasmKeyByExpBlock retrieves the keys of the stored Wasms that will expire at the given block height.
+// The method iterates over the WasmExp KeySet using the provided context and expiration block height,
+// and returns the keys as [][]byte.
+//
+// The method takes a sdk.Context and a uint64 value representing the block height at which expiration occurs.
+// It returns a [][]byte representing the keys of the stored Wasms and an error.
+//
+// Example usage:
+//
+//	wasmKeys, err := keeper.WasmKeyByExpBlock(ctx, expHeight)
+//	if err != nil {
+//	    return err
+//	}
+//	for _, key := range wasmKeys {
+//	    fmt.Println(string(key))
+//	}
+//
+// Note: The returned keys do not maintain an order.
+func (k Keeper) WasmKeyByExpBlock(ctx sdk.Context, expBlock int64) ([][]byte, error) {
```

Comment:
After taking a look to other keepers, in our repo and in Cosmos SDK modules, I would call this function something like `GetExpiredWasmKeys`.

Reply:
in this commit: [9cd4dcd](https://github.com/sedaprotocol/seda-chain/pull/266/commits/9cd4dcdfb9578b193a3adce22a0095e8ab17c507)